shared/selene/api/pantacor.py

```diff
@@ -165,3 +165,20 @@ def _call_pantacor_api(method: str, endpoint: str, **kwargs):
         )
 
     return response_data
+
+
+def update_pantacor_config(old_config: dict, new_config: dict):
+    """Make calls to the Pantacor API to update any values that have changed.
+
+    :param old_config: the config values before the update.
+    :param new_config: the new config values (which may be the same as the old values)
+    """
+    for config_name, new_value in new_config.items():
+        old_value = old_config[config_name]
+        if old_value != new_value:
+            if config_name == "auto_update":
+                _change_pantacor_update_policy(old_config["pantacor_id"], new_value)
+            elif config_name == "release_channel":
+                _change_pantacor_release_channel(old_config["pantacor_id"], new_value)
+            elif config_name == "ssh_public_key":
+                _change_pantacor_ssh_key(old_config["pantacor_id"], new_value)
```

Comment:
I imagine that we could collate these into single API calls if they're hitting the same endpoint in the future.

Reply:
It would be nice if we could.  That would be up to Pantacor to change their API.